### ISSUE TITLE
Timeout while waiting on state(s) (<PlayerState.PLAYING: 'playing'>,)

### ISSUE THREAD

**Issue by Mariusthvdb** [music-assistant/hass-music-assistant#680]
### What version of Music Assistant has the issue?

2022.7.0b7

### The problem

when several tts messages in sequence (queued) should be played, this message is logged, after the first message is played successfully

I can hear the original Google pop sound, then the tingling, but after that the message is not played.

### How to reproduce

have several device-trackers leave home, and automation trigger the tts message on state change of those trackers

### Relevant log output
```
Logger: music_assistant.players
Source: /usr/local/lib/python3.10/site-packages/music_assistant/models/player_queue.py:368 
First occurred: 12:15:55 (2 occurrences) 
Last logged: 13:26:18

Error while playing announcement
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/music_assistant/models/player_queue.py", line 365, in play_announcement
    await self._wait_for_state(PlayerState.PLAYING, silence_item.item_id)
  File "/usr/local/lib/python3.10/site-packages/music_assistant/models/player_queue.py", line 867, in _wait_for_state
    raise TimeoutError(f"Timeout while waiting on state(s) {state}")
TimeoutError: Timeout while waiting on state(s) (<PlayerState.PLAYING: 'playing'>,)
```

### Additional information

```
script:
  intercom_presence:
    alias: Intercom presence
    mode: queued
    sequence:
      - condition: state
        entity_id: input_boolean.intercom_presence
        state: 'on'
      - condition: >
          {{not is_state('input_select.activiteit','Home theater')}}
      - condition: >
          {{states(states('sensor.intercom')) not in ['unavailable','unknown']}}
      - service: media_player.volume_set
        data:
          entity_id: >
            {{state_attr('sensor.intercom','entity_id')}}
          volume_level: >
            {{states('input_number.intercom_volume')|float}}
      - service: tts.cloud_say
        data:
          language: >
            {{states('sensor.tts_language')}}
          options:
            gender: >
              {{states('input_select.tts_gender')|lower}}
          entity_id: >
            {{state_attr('sensor.intercom','entity_id')}}
          message: >
            {% set language = states('input_select.intercom_language') %}
            {% if language == 'En' %} {{message_en}}
            {% elif language == 'Nl' %} {{message_nl}}
            {% else %} {{message_en}}
            {% endif %}


automation:

  - alias: Presence Tracking
    id: presence_tracking
    mode: queued
    trigger:
      platform: state
      entity_id:
        - person.1
        - person.2
        - person.3
        - person.4
        - person.5
        - person.marijn

...
    action:
      - service: script.intercom_presence
        data:
          message_nl: >
            {# {% set entityid = trigger.entity_id %} #}
            {% set tostate = trigger.to_state.state %}
            {% set fromstate = trigger.from_state.state %}
            {% set zones = states.zone|map(attribute='name')|list %}
            {% set name = trigger.to_state.name %}

              Hi, op {{now().strftime('%X')}},
            {% if tostate == 'not_home' and fromstate in zones and fromstate != 'home' %}
              vertrok {{name}} van {{fromstate}}
            {% elif tostate == 'not_home' and fromstate not in zones and fromstate == 'home' %}
              vertrok {{name}} van {{fromstate}}
            {% elif tostate == 'not_home' and fromstate == 'driving' %}
              stopte {{name}} met rijden
            {% elif tostate == 'home' and fromstate == 'driving' %}
              stopte {{name}} met rijden en kwam thuis.

            {% elif fromstate == 'not_home' and tostate == 'driving' %}
              vertrok {{name}} en zit nu in de wagen

            {% elif tostate == 'driving' %}
              vertrok {{name}} van {{fromstate}} en zit nu in de wagen
            {% elif fromstate == 'not_home' %}
              kwam {{name}} aan bij {{tostate}}
            {% elif fromstate == 'driving' and tostate in zones %}
              stopte {{name}} met {{fromstate}} en kwam aan bij {{tostate}}
            {% else %}
              vertrok {{name}} van {{fromstate}} en kwam aan bij {{tostate}}
            {% endif %}
          message_en: >
            etcetc
```

### What version of Home Assistant Core are your running

2022.7.7

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Generic x86-64 (e.g. Intel NUC)

**Comment by erkr** [comment]
That sounds like a challenging use case for TTS. Thanks for the extensive additional info!

**Comment by Mariusthvdb** [comment]
well, it has been working for over 5 years now ;-) not really challenging actually.

In fact, its a simple automation triggering on my family's devices, and a simple script stating who left and arrived where...

It could be that the place still was in playing state when the second trigger happened. My daughter and wife left the premises together, so their trackers changed state practically at the same time.

As you can see the automation is `mode: queued` for that. I guess MA doesnt yet use that?

**Comment by marcelveldt** [comment]
1) You can remove the volume service call in your script now that we have an additional entity for that which sets the volume for you....

2) I was under the impression I guarded the code for this scenario but looks like I missed a spot. Normally it should execute only one announcement at a time. I will take a look at that.

3) mode: queued is correct. HA should not fire the announcements at the same time or only 1 will win. Somehow it looks like its not awaiting the announcement now so even the queued execution fails?

**Comment by marcelveldt** [comment]
Fix available in the 2022.7 release!
Please reopen (or create a new issue) if you still may experience issues. Thanks!
